enshrine-esdt-safe/src/from_sovereign/transfer_tokens.rs

```diff
@@ -52,7 +181,10 @@ pub trait TransferTokensModule:
         let mut output_payments = ManagedVec::new();
 
         for operation_token in operation_tokens.iter() {
-            if !self.has_sov_token_prefix(&operation_token.token_identifier) {
+            if !self.has_sov_prefix(
```

Comment:
You have to use `self.has_prefix` here because you can mint any token with prefix